### DISCUSSION TITLE
Worked without issues

### DISCUSSION THREAD

**Discussion by FroggMaster** [rcambrj/sovol-dryer-firmware#3] [Reports of success / failure]
* Dryer brand & model: SOVOL SH01
* Chip identifier: HC32F005x6xx
* USB to serial adapter: Flipper Zero

No additional drivers required. 
Added pull request with Flipper Zero Pin Out.

**Comment by rcambrj** [comment]
Thanks for the report and for the PR (https://github.com/rcambrj/sovol-dryer-firmware/pull/2)

Glad you were able to improve your device 🙂
